api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -3294,4 +3294,12 @@ public void getPatientIdentifiersByPatientProgram_shouldGetPatientsByIdentifierB
 		assertEquals(patientIdentifier.iterator().next().getIdentifier(), "XXXCCCAAA11");
 	}
 
+	@Test
+	public void getIdentifierValidator_shouldThrowPatientIdentifierExceptionWhenClassNotFound() throws Exception {
+		String invalidClassName = "com.example.InvalidIdentifierValidator";
+		String expectedErrorMessage = "Could not find patient identifier validator com.example.InvalidIdentifierValidator";
```

Comment:
Because these are used only once, do we even need variables for them?

Reply:
@dkayiwa I remove these variables.